tests/simulator/test_initialization.py

```
@@ -601,25 +558,32 @@
+            {},
             "particles",
+            diag_outputs=f"ppc/{ndim}/{interp_order}/{ddt_test_id}",
+            nbr_part_per_cell=ppc,
+            **kwargs,
         )
 
-        for patch in datahier.level(0).patches:
+        if cpp.mpi_rank() > 0:
+            return
+
+        for pi, patch in enumerate(datahier.level(0).patches):
```

Ruff

Loop control variable `pi` not used within loop body

Rename unused `pi` to `_pi`

(B007)